src/components/Node/Node.module.scss

```diff
@@ -10,7 +10,7 @@
   justify-content: center;
 
```

Comment:
The styles in the node header don't quite match the design. To fix, add the following to `.node { ... }`:
```css
font-size: 1.2em;
white-space: nowrap;
```